packages/client-sdk-nodejs/src/internal/cache-data-client.ts

```diff
@@ -228,14 +229,13 @@ export class CacheDataClient implements IDataClient {
         .getChannel()
         .watchConnectivityState(currentState, deadline, (error?: Error) => {
           if (error) {
-            this.logger.error(
-              `Unable to eagerly connect to Momento. Please contact Momento if this persists. currentState: ${currentState}, errorName: ${
-                error.name
-              } : errorMessage: ${error.message}, errorStack: ${
-                error.stack ? error.stack : 'Stack trace undefined'
-              }`
-            );
-            resolve();
+            const errorMessage = `Unable to eagerly connect to Momento. Please contact Momento if this persists. currentState: ${currentState}, errorName: ${
+              error.name
+            } : errorMessage: ${error.message}, errorStack: ${
+              error.stack ? error.stack : 'Stack trace undefined'
+            }`;
+            this.logger.error(errorMessage);
+            reject(new Error(errorMessage));
```

Comment:
will this be the same error that we rejected with earlier? if so, can we just do `reject(error)`? If not, can we use one of our SDK error types here too? (or create a new one, I think I did that for the .NET SDK.)